docs/hub/datasets-argilla.md

```diff
@@ -0,0 +1,110 @@
+# Argilla
+
+Argilla is a collaboration tool for AI engineers and domain experts who need high-quality outputs, full data ownership, and overall efficiency in their work.
```

Comment:
```suggestion
Argilla is a collaboration tool for AI engineers and domain experts who need to build high quality datasets for their projects.
```

Reply:
I know we have this in the readme but I think we can use a more direct and less salesy